connector_importer/models/recordset.py

```diff
@@ -175,6 +175,11 @@ def _get_json_from_binary(self, binary_data):
 
     def get_report(self):
         self.ensure_one()
+        # We need to invalidate cache because the context dict
+        # bin_size=False triggers the _compute_datas(self) method
+        # which has the @api.depends_context('bin_size') decorator.
+        # Flush all pending computations and updates to the database.
+        self.env.invalidate_all(flush=True)
```

Comment:
this sounds too much. Are you sure you cannot limit this only to the relevant models and fields?

Reply:
Indeed it was an overshoot. I ended up adding self.env["ir.attachment"].invalidate_model() to  the _set_serialized method before the self.invalidate_recordset((fname,))
https://github.com/OCA/connector-interfaces/pull/143/commits/328e580dd81d1aea3dd86747af51507fd5f6e90f